docs/ldml/tr35-general.md

```diff
@@ -642,6 +643,22 @@ When quotations are nested, the quotation marks and alternate marks are used in
 <alternateQuotationEnd>’</alternateQuotationEnd>
 ```
 
+### 4.1 <a name="Tailor_Linebreak_With_Delimiters" href="#Tailor_Linebreak_With_Delimiters">Tailoring Linebreak Using Delimiters</a>
+
+The delimiter data can be used for language-specific tailoring of linebreak behavior, as suggested
+in the [description of linebreak class QU: Quotation](https://www.unicode.org/reports/tr14/#QU)
+in [[UAX14](https://www.unicode.org/reports/tr41/#UAX14)]. This is an example of
+[tailoring type](https://www.unicode.org/reports/tr14/#Tailoring) 1 (from that same document),
+changing the line breaking class assignment for some characters.
+
+If the values of `<quotationStart>` and `<quotationEnd>` are different, then:
+* if the value of `<quotationStart>` is a single character with linebreak class QU: Quotation, change its class to OP: Open Punctuation.
+* if the value of `<quotationEnd>` is a single character with linebreak class QU: Quotation, change its class to CL: Close Punctuation.
+Similarly for `<alternateQuotationStart>` and `<alternateQuotationEnd>`.
+
+Some characters with multiple uses should generally be excluded from this linebreak class remapping, such as:
```

Comment:
Wouldn't you instead say that if unmatched then that character would be excluded. And then provide the two possible matching characters.

Reply:
Not sure what you mean. What I am trying to say is we should always leave U+2019 as QU and not remap to CL.

Reply:
OK, expanded the exceptions info.